src/components/dropdown/dropdown.sandbox.html

```diff
@@ -2,323 +2,637 @@
+            <m-dropdown-item value="SY"
+                             label="SYRIENNE, RÉPUBLIQUE ARABE"></m-dropdown-item>
+            <m-dropdown-item value="TJ"
+                             label="TADJIKISTAN"></m-dropdown-item>
+            <m-dropdown-item value="TW"
+                             label="TAÏWAN, PROVINCE DE CHINE"></m-dropdown-item>
+            <m-dropdown-item value="TZ"
+                             label="TANZANIE, RÉPUBLIQUE-UNIE DE"></m-dropdown-item>
+            <m-dropdown-item value="TD"
+                             label="TCHAD"></m-dropdown-item>
+            <m-dropdown-item value="CZ"
+                             label="TCHÈQUE, RÉPUBLIQUE"></m-dropdown-item>
+            <m-dropdown-item value="TF"
+                             label="TERRES AUSTRALES FRANÇAISES"></m-dropdown-item>
+            <m-dropdown-item value="TH"
+                             label="THAÏLANDE"></m-dropdown-item>
+            <m-dropdown-item value="TL"
+                             label="TIMOR-LESTE"></m-dropdown-item>
+            <m-dropdown-item value="TG"
+                             label="TOGO"></m-dropdown-item>
+            <m-dropdown-item value="TK"
+                             label="TOKELAU"></m-dropdown-item>
+            <m-dropdown-item value="TO"
+                             label="TONGA"></m-dropdown-item>
+            <m-dropdown-item value="TT"
+                             label="TRINITÉ-ET-TOBAGO"></m-dropdown-item>
+            <m-dropdown-item value="TN"
+                             label="TUNISIE"></m-dropdown-item>
+            <m-dropdown-item value="TM"
+                             label="TURKMÉNISTAN"></m-dropdown-item>
+            <m-dropdown-item value="TC"
+                             label="TURKS ET CAÏQUES, ÎLES"></m-dropdown-item>
+            <m-dropdown-item value="TR"
+                             label="TURQUIE"></m-dropdown-item>
+            <m-dropdown-item value="TV"
+                             label="TUVALU"></m-dropdown-item>
+            <m-dropdown-item value="UA"
+                             label="UKRAINE"></m-dropdown-item>
+            <m-dropdown-item value="UY"
+                             label="URUGUAY"></m-dropdown-item>
+            <m-dropdown-item value="VU"
+                             label="VANUATU"></m-dropdown-item>
+            <m-dropdown-item value="VE"
+                             label="VENEZUELA"></m-dropdown-item>
+            <m-dropdown-item value="VN"
+                             label="VIET NAM"></m-dropdown-item>
+            <m-dropdown-item value="WF"
+                             label="WALLIS ET FUTUNA"></m-dropdown-item>
+            <m-dropdown-item value="YE"
+                             label="YÉMEN"></m-dropdown-item>
+            <m-dropdown-item value="ZM"
+                             label="ZAMBIE"></m-dropdown-item>
+            <m-dropdown-item value="ZW"
+                             label="ZIMBABWE"></m-dropdown-item>
```

Comment:
ça n'a pas rapport avec le PR, mais ça aurait été possible de générer cette grosse liste via un v-for sur une liste d'objets au lieu de le faire dans le html?